client/src/docs-ui/select-anchor/reroute-if-necessary.worker.ts

```diff
@@ -0,0 +1,21 @@
+import {parseURL} from "../../utils/url/url.worker";
+
+const rereouteCache = new Map<string, string>();
+
+export const rerouteIfNecessary = async (path: string) => {
+  if (rereouteCache.has(path)) {
+    return rereouteCache.get(path);
+  }
+  const rerouted = await (async () => {
+    if (
+      path.includes("/lib") &&
+      (path.includes("/q/platform/ios") || path.includes("/q/platform/android"))
```

Comment:
What exactly is this function doing? It seems strange to hardcode these values here when they are constants in a FILTER enum that we maintain.

Reply:
Interesting. Not sure if this is needed anymore. I believe it was originally responsible for rerouting to sdk as the default. Then someone must have swapped out the "/sdk" portions of these strings with "/lib" when it went GA. Unless you can think of a reason not to, I'll get rid of this dud code.